README.md

```diff
@@ -0,0 +1,216 @@
+<a name="#readme-top"></a>
```

Comment:
- [ ] I couldn't find a working backend app for this project, so much of the functionality is not present at the time of this review, as you kindly noted in your review request.

note: this means I couldn't test the core functionality of the app or the code itself.  These items will be taken a look at in the next review.